core/src/com/mygdx/crazybomber/model/player/Player.java

```diff
@@ -114,14 +114,22 @@ public Bomb dropBomb() {
         } else {
             droppedBomb.setXCoordinate((int) Math.round(getYCoordinate()));
         }
+        ScheduledExecutorService _scheduledExecutorService = Executors.newScheduledThreadPool(5);
```

Comment:
Doesnt this create 5 threads every time a bomb is dropped? why 5 and you should prob make this a class variable

Reply:
made it 1 threat at droptime